cognition_objects/markdown_file.py

```diff
@@ -0,0 +1,116 @@
+from datetime import datetime
+from typing import Dict, List, Optional, Tuple
+
+from ..cognition_objects import message
+from ..business_objects import general
+from ..session import session
+from ..models import CognitionMarkdownFile
+from .. import enums
+from sqlalchemy import func, alias, Integer
+from sqlalchemy.orm import aliased
+
+
+def get(md_file_id: str) -> CognitionMarkdownFile:
+    return (
+        session.query(CognitionMarkdownFile)
+        .filter(CognitionMarkdownFile.id == md_file_id)
+        .first()
+    )
+
+
+def get_all_paginated_for_category_origin(
+    org_id: str,
+    category_origin: str,
+    page: int,
+    limit: int,
+) -> Tuple[int, int, List[CognitionMarkdownFile]]:
+    total_count = (
+        session.query(func.count(CognitionMarkdownFile.id))
+        .filter(CognitionMarkdownFile.organization_id == org_id)
+        .scalar()
+    )
```

Comment:
```suggestion
    total_count = (
        session.query(CognitionMarkdownFile.id)
        .filter(CognitionMarkdownFile.organization_id == org_id)
        .count()
    )
```
I think this improves readability